modules/Accounts/language/nl_nl.lang.php

```diff
@@ -208,5 +208,12 @@
 	'Projects' => 'Projecten',
 	'Timecontrol' => 'Urenregistratie',
 	'InventoryDetails' => 'Voorraad Details',
+	'Exact VAT Code for Account' => 'BTW code voor klant',
+	'Is Converted From Lead' => 'Is gemaakt vanuit lead',
+	'Converted From Lead' => 'Gemaakt vanuit lead',
+	'Exact Payment Condition for Account' => 'Exact betalingsconditie voor klant',
+	'' => '#leeg#',
```

Comment:
Are these three correct. I think it will be better if you don't leave empty lines like that just in case.